src/main.js

```diff
@@ -145,57 +148,67 @@ let load = base.loader({
     },
   },
 
-  queue: {
-    requires: ['cachemanagers', 'cfg', 'sqs', 'profile', 'monitor'],
-    setup: async ({cachemanagers, cfg, sqs, profile, sqsQueues, monitor}) => {
+  queueUrl: {
+    requires: ['cfg', 'sqs', 'profile'],
+    setup: async ({cfg, sqs, profile}) => initQueue(sqs, `cloud-mirror-${profile}`),
+  },
 
-      let handler = async (obj) => {
-        assert(obj.id, 'must provide id in queue request');
-        assert(typeof obj.id === 'string', 'id must be string');
-        assert(obj.url, 'must provide url in queue request');
-        assert(typeof obj.url === 'string', 'url must be string');
+  queueFactory: {
```

Comment:
There's still the downside that we create handlers in the process that's only supposed to be putting things into the queue.  This isn't a huge deal because we never actually call `.start()` on the queue's that are used for inserting.